web/sites/default/config/user.role.administrator.yml

```diff
@@ -1,11 +1,18 @@
 uuid: 77a4231f-7732-436c-830d-09cbb51e65a4
 langcode: en
 status: true
-dependencies: {  }
+dependencies:
+  module:
+    - field_permissions
 _core:
   default_config_hash: 2EL4nOaSWrCYL11npXSGwLjX_DJoj2riIUqGaNyZFWU
 id: administrator
 label: Administrator
 weight: 8
 is_admin: true
-permissions: {  }
+permissions:
+  - 'create field_group_path'
```

Comment:
This change should be unnecessary since the admin role already has all permissions.